modules/utility/init.zsh

```diff
@@ -175,9 +175,13 @@ elif (( $+commands[wget] )); then
   alias get='wget --continue --progress=bar --timestamping'
 fi
 
-# Resource Usage
-if (( $+commands[pydf] )); then
-  alias df=pydf
+if zstyle -s ':prezto:module:utility' pydf 'yes'; then
```

Comment:
* Usually, we prefer `zstyle -t` or `zstyle -T` for boolean
* Would you consider `use-pydf` instead of `pydf`?
* We'll also need doc for this in `utility/README.md` and in default `runcoms/depreztorc` with the default commented out.